+nosnoc/+dcs/Stewart.m

```diff
@@ -11,12 +11,13 @@
         z_all % CasADi symbolic variable - all algebraic variables (user provided and Stewart DCS specific)
 
         f_x  % CasADi symbolic expression -  r.h.s. of the ODE, f_x = sum_i F_i*theta_i , i is the index of the subystems
-        g_Stewart % CasADi symbolic expression - TODO: this is same as g_ind? maybe have consistent names g_ind and g_ind_sys?
+        g_ind % CasADi symbolic expression - TODO: this is same as g_ind? maybe have consistent names g_ind and g_ind_sys?
```

Comment:
one would think maybe also of g_ind(ex), write g_indicator and give refernece to stewart indicator functions as we called them in the original fesd paper